### DISCUSSION TITLE
Authentication difference between users and admins

### DISCUSSION THREAD

**Discussion by richardltc** [pocketbase/pocketbase#169] [Q&A] [answered]
Hi,

I see that there are two different ways of authenticating users, depending on whether they are admins or not. Would the user have to specify that they were an admin, in order for the correct authentication method to be used?

Richard

**Comment by ganigeorgiev** [comment, marked as answer]
**Admins** and **Users** are two different things in PocketBase - each have their own separate table and different business logic behind them.

**Admins** are basically the developer accounts. All admins act as "super-admin" since **there are no permission rules for them**, aka. they can read, modify and delete everything (including other admins). Admins can login only via email/password.

**Users** represents your application level users. They can take part in the Collection API rules and supports different authentication methods (currently email/password and OAuth2). Depending on your use case, **Users** can have role/group/team if you attach to them a custom field(s) (see #165).

**Reply by richardltc** [reply]
Thanks for the clarification

**Reply by janzheng** [reply]
I'm currently switching between Admin and User to develop my app — is there any way to see my "current user / logged in state" to know what I'm allowed to do? I have different browsers open w/ different users and it often gets confusing

**Reply by ganigeorgiev** [reply]
@janzheng If you are using the [Javascript](https://github.com/pocketbase/js-sdk) or [Dart](https://github.com/pocketbase/dart-sdk) sdk you can inspect your current logged in user model by accessing `client.authStore.model`.

I'm a little confused what do you mean by _"switching between Admin and User to develop my app - is there any way to see my "current user / logged in state" to know what I'm allowed to do"_. As mentioned in the comment above, Users and Admins are 2 separate entities and serves different purposes. If you are looking for something like "show me a list of permissions for my user" - there is no such thing since this depend on your collection API rules (which act also as filter).

Could you elaborate a little more on your use case (or open a new discussion)?
